source/core/GenericFunctions.cpp

```diff
@@ -1667,10 +1668,6 @@ VIREO_FUNCTION_SIGNATURET(ClusterBinaryOp, AggregateBinOpInstruction)
             convertSnippet->_p1 = (AQBlock1*)(intptr_t(whichConvertArg) << 24);
             snippet = (Instruction4<AQBlock1, AQBlock1, AQBlock1, AQBlock1>*) next;
         }
```

Comment:
This is wrong to fall through.  It will now handle convertArgs for every OTHER cluster elemenet.
It also now needs to save/restore the whichConvertArg (convertSnippet->_p1) for each element, not just the first.